source/fundamentals/connection.txt

```diff
@@ -25,13 +25,13 @@ example explains each part of a sample connection URI:
 .. figure:: /includes/figures/connection_string_parts.png
    :alt: Each part of the connection string
 
-In this example, we use ``mongodb+srv`` for the protocol, which specifies the
-:manual:`DNS Seedlist Connection Format
-</reference/connection-string/#std-label-connections-dns-seedlist>`. This indicates
-that the URI's hostname corresponds to the DNS SRV record of your
-MongoDB instance or deployment. If your instance or deployment does not have a
-DNS SRV record, use ``mongodb`` to specify the :manual:`Standard Connection
-Format </reference/connection-string/#standard-connection-string-format>`.
+In this example, we use ``mongodb`` for the protocol, which specifies the
+:manual:`Standard Connection String Format
+</reference/connection-string/#std-label-connections-standard-connection-string-format>`.
+You can also use the :manual:`DNS Seed List Connection Format
+</reference/connection-string/#dns-seed-list-connection-format>` if you
+want more flexibility of deployment and the ability to change the
+servers in rotation without reconfiguring clients.
```

Comment:
Nice! I see the image was updated to use the `mongodb` protocol and the `maxPoolSize` / `w` options. The [stage link](https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-13828-connection-guide/fundamentals/connection/) in the description shows the old image. Is that link out of date? It looks like the text content is updated.

Reply:
Thanks for catching that! I had referenced an incorrect path.